.gitignoresource

```diff
@@ -0,0 +1 @@
+export SENDGRID_API_KEY='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' sendgrid.env ./sendgrid.env
```

Comment:
Can you explain what this API key is for? Is there a way to use the twilio API without including the API key directly in code? The risk here is that somebody could find the code on Github, copy the API key, and start sending requests pretending to be your project and use up all the quota.

Reply:
Chnaged form epxlictly using the API key to uaing envromnet variables

Reply:
Did you generate a new key?

Reply:
ping

Reply:
No i did not request a new key